cli/src/pcluster/models/common.py

```diff
@@ -245,14 +245,20 @@ def _download_s3_objects_with_prefix(self, task_id, destdir):
             os.remove(compressed_path)
 
 
-def export_stack_events(stack_name: str, output_file: str):
-    """Save CFN stack events into a file."""
+def get_all_stack_events(stack_name: str):
+    """Retrieve all stack events."""
     stack_events = []
     chunk = AWSApi.instance().cfn.get_stack_events(stack_name)
     stack_events.append(chunk["StackEvents"])
     while chunk.get("nextToken"):
         chunk = AWSApi.instance().cfn.get_stack_events(stack_name, next_token=chunk["nextToken"])
         stack_events.append(chunk["StackEvents"])
+    return stack_events
```

Comment:
[Question] Why do we realize this into a list and not return it as a generator?

Reply:
The function is used in two places. Forcing it at this level can reduce code to make it in the list in two places